swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+import uuid
+
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0):
+        self.id = id if id is not None else uuid.uuid4().int
+        self.condition = condition
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}."
+
+    
+    def get_category(self):
+        return "Item"
+        # class_name = __class__.__name__
+        
+    def condition_description(self):
+        condition_status = ["bad", "heavily used", "acceptable", "good", "like new", "new"]
+        return condition_status[self.condition]
```

Comment:
I like that you used a datatype to hold the condition strings and then use `self.condition`. This allows you to stay away from using if/elif/else conditional statements which would be places where bugs could get introduced in the future and keeps your code concise/readable!